net/yolov2/test.py

```diff
@@ -78,14 +79,27 @@ def postprocess(self, net_out, im, save = True):
 			if top   < 0    :   top = 0
 			if bot   > h - 1:   bot = h - 1
 			thick = int((h+w)/300)
-			cv2.rectangle(imgcv, 
-				(left, top), (right, bot), 
+			cv2.rectangle(imgcv,
+				(left, top), (right, bot),
 				colors[max_indx], thick)
 			mess = '{}'.format(label)
-			cv2.putText(imgcv, mess, (left, top - 12), 
+			cv2.putText(imgcv, mess, (left, top - 12),
 				0, 1e-3 * h, colors[max_indx],thick//3)
+			if self.FLAGS.json:
+				line = 	('{"label":"%s",'
+						'"topleft":{"x":%d,"y":%d},'
+						'"bottomright":{"x":%d,"y":%d}},\n') % \
+						(mess, left, top, right, bot)
+				textBuff += line
 
+	# Removing trailing comma+newline adding json list terminator.
+	textBuff = textBuff[:-2] + "]"
 	if not save: return imgcv
-	outfolder = os.path.join(self.FLAGS.test, 'out') 
+	outfolder = os.path.join(self.FLAGS.test, 'out')
 	img_name = os.path.join(outfolder, im.split('/')[-1])
+	if self.FLAGS.json:
```

Comment:
Please bring this `if` block to before line `97` and add a `return` to the end of it.